src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,44 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+     final int EMPTY_STACK_VALUE=-1;
+     final int STACK_CAPACITY =12;
+    private final int[] numbers = new int[STACK_CAPACITY];
+  
+    public int total = EMPTY_STACK_VALUE;
+
+
+    protected void countIn(int in) {
+        if (!isFull())
+            numbers[++total] = in;
+    }
+
+
+    public boolean callCheck() {
+        return total == EMPTY_STACK_VALUE;
```

Comment:
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you used review hints ('Github - PR checkpoint' at wikamp).
This constant should be different (separate) than the one in peekaboo and countOut methods.